pkg/apis/core/v1/defaults.go

```diff
@@ -96,6 +96,20 @@ func SetDefaults_Container(obj *v1.Container) {
 	}
 }
 
+func SetDefaults_EphemeralContainer(obj *v1.EphemeralContainer) {
+	if obj.ImagePullPolicy == "" {
```

Comment:
I think we want the same defaulting for EphemeralContainer as for Container, could you instead invoke `SetDefaults_Container()` on `EphemeralContainerCommon`?

```
SetDefaults_Container((*v1.Container)(&obj.EphemeralContainerCommon))
```